deltachat-rpc-client/src/deltachat_rpc_client/rpc.py

```diff
@@ -60,7 +59,6 @@ async def method(*args, **kwargs):
             }
             data = (json.dumps(request) + "\n").encode()
             self.process.stdin.write(data)
-            event = asyncio.Event()
```

Comment:
@link2xt this is not used but I don't know if the real issue is that it actually should be used? reading the comment in `self.request_events` initialization it seems that no